Feat/release cd (#315)
inrupt/solid-client-js: .github/pull_request_template.md

- [ ] The changelog has been updated, if applicable.
- [ ] New functions/types have been exported in `index.ts`, if applicable.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

<!-- When cutting a release: -->

This PR bumps the version to <version number>.

# Checklist

> - [ ] I used `npm version` tu update `package.json`

Reviewer: ```suggestion
- [ ] I used `npm version` to update `package.json`, inspecting the changelog to determine if the release was major, minor or patch.
```

> - [ ] The CHANGELOG has been updated to show version and release date

Reviewer: ```suggestion
- [ ] The CHANGELOG has been updated to show version and release date - https://keepachangelog.com/en/1.0.0/.
```

> - [ ] The **only** commits in this PR are
>   - the CHANGELOG update
>   - the version update
> - [ ] I make sure **not** to squash these commits
> - [ ] Once this PR is merged, I will push the tag created by `npm version` (e.g.)

Reviewer: ```suggestion
- [ ] The **only** commits in this PR are
  - the CHANGELOG update.
  - the version update.
- [ ] I make sure **not** to squash these commits.
- [ ] Once this PR is merged, I will push the tag created by `npm version`.
```

Periods at the end everywhere, for pedantic consistency. Were you planning to add anything to `(e.g.)`?
